samples/excel/tables/create-table.yaml

```diff
@@ -0,0 +1,94 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Create a table
+description: Creates a table with four columns and seven rows.
+script:
+  content: |+
+    $('#create-table').click(createTable);
+
+    /* Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
```

Comment:
I don't think you need to define the sheet variable outside of Excel.run. You can remove this and make line 17 be let sheet = ...